### PR TITLE
feat: home page

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the landing page of the quiz app using the maple UI component library.
Screenshot (457)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,44 @@
...
+                    <div class="para-div">
+                        <p class="paragraph ">Let's have some fun playing quiz</p>
+                    </div>
+                    <div class="para-button">
+                        <a href="./quiz-cate.html" class="btn quiz-btn-background-color ">See Categories</a>
+                        <a href="./quiz-rules.html" class="btn quiz-btn-sec">Rules</a>
+                    </div>
+                </div>
+                <div class="hero-img-div">
+                    <img class="hero-img img-res shadow" src="./Assests/think-final.png" alt="">
```
Comment (vivekmehto) [inline index.html]: you should always mention **alt** attribute of image

File: index.html [hunk 2]
```diff
@@ -4,9 +4,44 @@
...
     <title>Document</title>
 </head>
 <body>
-    
+    <nav class="nav-bar">
+        <div class="nav-left">
+            <a class=" txt-xl semibold-font header-text-color" href="#">Maple Quiz</a>
+        </div>
+        <div class="nav-right">
+            <a href="" class=" txt-lg header-text-color">Docs</a>
```
Comment (vivekmehto) [inline index.html:17]: docs in quiz app?

File: CSS/quiz-home.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    height: 20rem;
+}
+.hero-img{
+    width: 30rem;
+}
+.footer-text{
+    color: var(--headline-color);
+}
+.nav-bar{
+    padding: 1rem 3rem 1rem 3rem;
```
Comment (vivekmehto) [inline CSS/quiz-home.css]: we can use the shorthand property ``padding: top/bottom | left/right``

**Review by vivekmehto — COMMENTED** [review]
few suggestions and you can make flex utility class and use it because flex is repeating 4-5 times. Apart from that everything looks good.

**Review by RahulSingh9131 — APPROVED** [review]
Everything looks fine to me. Vivek had already pointed out some changes. Approved from my side.

File: index.html (hunk 2 above)
Comment (krishanu2911) [reply index.html:17]: That was just for sample will update it thanks for pointing out

File: index.html (hunk 1 above)
Comment (krishanu2911) [reply index.html]: already updated in the above commit.

File: CSS/quiz-home.css (hunk 3 above)
Comment (krishanu2911) [reply CSS/quiz-home.css]: oh ya i just forget to use that thanks for pointing out updated

**Comment by krishanu2911** [comment]
> few suggestions and you can make flex utility class and use it because flex is repeating 4-5 times. Apart from that everything looks good.

Done forget to use my already defined class in maple UI (CL)

**Review by vivekmehto — COMMENTED** [review]
everything looks good clean code.

**Review by vivekmehto — APPROVED** [review]

**Pull request merged**